docs/procedures.md

```diff
@@ -50,30 +48,25 @@ Execute the following commands on your local machine, not production.
 
     Pushes to your `origin` remote are optional.
 
-4.  Define which version you're going to release (replace X and Y with correct numbers):
-
-        $ AMY_CURRENT=v2.X.Y-dev  # this needs to correspond to what you have in amy/workshops/__init__.py and package.json
-        $ AMY_VERSION=v2.X.Y
-        $ AMY_NEXT_VERSION=v2.X+1.0-dev
-
-5.  Merge `develop` into `main` branch (be careful, as there are sometimes conflicts that need to be manually resolved):
+4.  Merge `develop` into `main` branch (be careful, as there are sometimes conflicts that need to be manually resolved):
 
         $ git checkout main
         $ git merge --no-ff develop
 
-6.  Bump version on `main`:
+5.  Bump version on `main` (stable version corresponding to the milestone):
 
-        $ make bumpversion CURRENT=$AMY_CURRENT NEXT=$AMY_VERSION
-        $ git add amy/workshops/__init__.py package.json
-        $ git commit -m "Bumping version to $AMY_VERSION"
+        $ # manually edit version in `amy/__init__.py` and `package.json`
+        $ # use stable version string, e.g. `"v3.3.0"`
```

Comment:
Not sure if you have this already, but it might not hurt to have a couple of notes about when to upgrade to a different version i.e. when to bump from 3.3.1 vs 3.4.0

Reply:
Thank you for the comment, @lauryndbrown. I added some notes on versioning, and fixed some instructions on what version should be used.